sponsors/index.html

```diff
@@ -42,7 +42,33 @@
     </script>
 
     <section id="sponsors">
-        <h1 align="center" class="content-header">Sponsors</h1>
+        <h1 align="center"  class="content-header">Sponsor Us!</h1>
+
+
+        <div align= "center" class="sponsor-call">
+            We are committed to cultivating diverse voices in our
+            student body while maintaining engagement between company representatives and our members in
+            whatever setting that may be. With your contribution, we will plan in-person and hybrid opportunities through
+            which we aim to help members gain confidence in pursuing their ambitions. In turn, you will be able to connect with these dedicated students and gain more recognition within our organization!
+        </div>
+
+        <div class="sponsor-call" style="padding-bottom: 0px;">
+          <p style="font-size: 20px;">Sponsorship Tiers:</p>  
```

Comment:
I think it would look better if this "Sponsorship Tier" text were a red header like the "Sponsors" text below it, but lmk what you think